libs/langchain/langchain/retrievers/kendra.py

```diff
@@ -178,6 +180,13 @@ def get_document_attributes_dict(self) -> Dict[str, DocumentAttributeValueType]:
         """Document attributes dict."""
         return {attr.Key: attr.Value.value for attr in (self.DocumentAttributes or [])}
 
+    def get_score_attribute(self) -> str:
+        """Document Score Confidence"""
+        if self.ScoreAttributes is not None:
+            return self.ScoreAttributes["ScoreConfidence"]
+        else:
+            return "NOT_AVAILABLE"
```

Comment:
Can we re-write this part to simply this:

```python
return self.ScoreAttributes.get("ScoreConfidence", "NOT_AVAILABLE")
```

Reply:
Good suggestion but it will not work as we have `ScoreAttributes: Optional[dict]`. ScoreAttributes can be None.
Getting error while linting.
error: Item "None" of "Optional[Dict[Any, Any]]" has no attribute "get"

Another approach we can do is
```
def get_score_attribute(self) -> str:
        """Document Score Confidence"""
        score_attributes = self.ScoreAttributes or {}
        return score_attributes.get("ScoreConfidence", "NOT_AVAILABLE")
```
It will have additional variable so I recommend to keep it same.